test/utils/helper.ts

```diff
@@ -1,15 +1,26 @@
 import { TextDocument } from 'vscode-languageserver-textdocument';
 import * as path from 'path';
 import { promises as fs } from 'fs';
+import { WorkspaceManager } from '../../src/services/workspaceManager';
+import { createConnection } from 'vscode-languageserver/node';
 
 export async function getDoc(filename: string): Promise<TextDocument> {
   const file = await fs.readFile(path.resolve('test', 'data', filename), {
     encoding: 'utf8',
   });
-  return TextDocument.create('uri', 'ansible', 1, file);
+  const docUri = path.resolve('test', 'data', filename).toString();
+  return TextDocument.create(docUri, 'ansible', 1, file);
 }
 
 export function isWindows(): boolean {
   // win32 applies to x64 arch too, is the platform name
   return process.platform === 'win32';
 }
+
+export function createTestWorkspaceManager(): WorkspaceManager {
```

Comment:
Could you add jsdoc here? It's useful especially for things that are exported because they are used in other places and it'd be helpful when using it in tests, for example.

Reply:
@webknjaz I was curious about the subject and ended up reading https://eslint.org/docs/rules/require-jsdoc and https://eslint.org/blog/2018/11/jsdoc-end-of-life -- I guess you will like playing with it and adding the missing comments?

I may also suggest reading https://code.visualstudio.com/docs/languages/typescript#_jsdoc-support as I hope that eslint plugin would have work with type-less jsdocs as we don't want to repeat ourselves.

Reply:
Thanks. I will add jsdoc for `createTestWorkspaceManager` function.

Reply:
> I guess you will like playing with it and adding the missing comments?

I think this could be a follow-up to some Sphinx autogeneration setup.